### PR TITLE
Solution template

### PR DESCRIPTION
**Pull request by Deepika-Karthi**
# Justification

 If this branch is merged with main it will be helpful in building the application with respect to this template.

# Implementation

Add template of the solution explorer.

### CODE REVIEW

File: ExampleService_WebApi - Copy/obj/ExampleService_WebApi.csproj.nuget.dgspec.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
```
Comment (jayabal-cf) [inline ExampleService_WebApi - Copy/obj/ExampleService_WebApi.csproj.nuget.dgspec.json]: We should not commit the binary files. Like the auto generated files inside `obj` and `bin` folders. 

We should add [.gitignore file](https://git-scm.com/docs/gitignore#:~:text=A%20gitignore%20file,specifies%20a%20pattern) to tell git to not to consider these files as changes.

File: ExampleService_WebApi - Copy/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "dotnet.defaultSolution": "ExampleService_WebApi.sln"
```
Comment (jayabal-cf) [inline ExampleService_WebApi - Copy/.vscode/settings.json]: Update the solution name here and add the `task.json` file also

**Review by jayabal-cf — CHANGES_REQUESTED** [review]

File: ExampleService_WebApi - Copy/obj/ExampleService_WebApi.csproj.nuget.dgspec.json (hunk 1 above)
Comment (Deepika-Karthi) [reply ExampleService_WebApi - Copy/obj/ExampleService_WebApi.csproj.nuget.dgspec.json]: Add git ignore file

File: ExampleService_WebApi - Copy/.vscode/settings.json (hunk 2 above)
Comment (Deepika-Karthi) [reply ExampleService_WebApi - Copy/.vscode/settings.json]: Add task.json file

File: ExampleService/ExampleService_WebApi/.vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
```
Comment (jayabal-cf) [inline ExampleService/ExampleService_WebApi/.vscode/tasks.json]: Should the .vscode folder be in the top level `ExampleService` folder?

File: ExampleService/ExampleService_WebApi/ExampleService_WebApi.sln [hunk 4]
```diff
@@ -0,0 +1,25 @@
+﻿
```
Comment (jayabal-cf) [inline ExampleService/ExampleService_WebApi/ExampleService_WebApi.sln]: Why do we need this solution?

**Review by jayabal-cf — CHANGES_REQUESTED** [review]

File: ExampleService/ExampleService_WebApi/.vscode/tasks.json (hunk 3 above)
Comment (Deepika-Karthi) [reply ExampleService/ExampleService_WebApi/.vscode/tasks.json]: Move to the top level.

File: ExampleService/ExampleService_WebApi/ExampleService_WebApi.sln (hunk 4 above)
Comment (Deepika-Karthi) [reply ExampleService/ExampleService_WebApi/ExampleService_WebApi.sln]: Remove the unused solution.

**Review by jayabal-cf — APPROVED** [review]

**Pull request merged**